project-koku/koku
# [COST-4012] Don't check serviceinfo2 for Azure HCS filter

## Jira Ticket

[COST-4012](https://issues.redhat.com/browse/COST-4012)

## Description

This change will remove the check of serviceinfo2 from the Azure filter as it is a legacy field and the information we care about is being matched via other criteria.

Opened an [MR to update the failing](https://gitlab.cee.redhat.com/insights-qe/iqe-cost-management-plugin/-/merge_requests/1690) smoke test as well since the data covered is different.

## Testing

1. Checkout Branch
2. Restart Koku
3. Create and Load the test customer data for all source types since the SQL was updated to remove the sqlsafes: `make create-test-customer` followed by `make load-test-customer-data`
4. Data should process fine and minio should have HCS files for Azure, AWS, and GCP.

## Notes

...
